api/test/integration/test_security_GET_endpoints.tavern.yaml

```
@@ -776,40 +776,54 @@ stages:
             - id: 1
               username: wazuh
               allow_run_as: false
-              roles: []
+              roles:
+                - 1
             - id: 2
               username: wazuh-wui
               allow_run_as: true
-              roles: []
+              roles:
+                - 2
```

```suggestion
                - 1
```
The user `wazuh-wui` has admin permissions, right?